include/databaseFunctions.php

```diff
@@ -57,39 +46,25 @@ function getNumberPosts(){
 		$result = dbQuery("
 				SELECT COUNT(postID)
 				FROM posts
+				WHERE isPic=0
 			")->fetch();
-		return $result;
+		return $result['COUNT(postID)'];
 	}
 
 
 	//PIC FUNCTIONS
-	function InsertPic($photographer, $date, $title, $link, $flavor){
+	function InsertPic($photographer, $title, $body, $link, $flavor){
 		$result = dbQuery("
-			INSERT INTO pics (photographer, date, title, link, flavor)
-			VALUES('$photographer', '$date', '$title', '$link', '$flavor')
+			INSERT INTO posts (author, title, body, isPic, link, flavor)
```

Comment:
This is a perfectly reasonable way to do this, although since you're moving pretty fast, I'll throw out an alternative.

Are you familiar with JSON encoding? You can take an object (like a PHP array) and turn it into a string. Sometimes it's tedious to have a separate MySQL field for every single thing you want to store about something, especially in this case where you won't need the link or flavor for normal blog posts (I think).

In this case, you could make a field called `ExtraData` or something like that, and take all that extra stuff, put it in an array, and JSON encode it. This is nice because it allows you to store all kinds of extra data in that field, even stuff you don't currently anticipate needing, and you won't need to change the db schema later. If you think you may ever want to support more than just blog posts and pictures, this'll be even more appropriate since the fields might be different for each record type.